src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions.Internal/FeatureFlags.cs

```diff
@@ -23,5 +23,7 @@ private static ICollection<string> CreateFeatures()
         }
 
         public static bool IsRequested(string name) => _features.Contains(name);
+
+        public static bool AnalyzeFormat() => _features.Contains("FORMAT");
```

Comment:
nit: rename to `IsAnalyzeFormatEnabled => _features.Contains("FORMAT")`

bool method/properties should start with `Is` or `Has` or something like that